tests/legacy-cli/e2e/assets/13.0-project/src/test.ts

```diff
@@ -23,4 +23,4 @@ getTestBed().initTestEnvironment(
 // Then we find all the tests.
 const context = require.context('./', true, /\.spec\.ts$/);
 // And load the modules.
-context.keys().map(context);
+context.keys().forEach(context);
```

Comment:
This file and the below are tests assets as are based on file generated with Angular CLI 12. These should not be modified.